bench/parser.ex

```diff
@@ -4,7 +4,7 @@ local_log_to_server_log = fn filename ->
   File.read!(filename)
   |> String.split("\n")
   |> Enum.map(fn
-    <<"L ", local_log::binary>> -> String.replace(local_log, ": ", ".000 - ")
+    <<"L ", local_log::binary>> -> Regex.replace(~r/(?<=:\d\d): /, local_log, ".000 - ")
```

Comment:
What is this matching? :D

Reply:
It matches any `: ` that is prefixed by `:00`. It takes a line like this: 

`L 12/30/2019 - 17:51:08: server_cvar: "cash_team_win_by_defusing_bomb" "3250"` 

and turns it into:

`12/30/2019 - 17:51:08.000 - server_cvar: "cash_team_win_by_defusing_bomb" "3250"` 

(note the `L ` is removed by the pattern match). This is just to make it possible to use "local" logs (logs written by the server directly not disk and not sent through HTTP).

Reply:
Should I add a comment to explain it?

Reply:
That's what I thought.. Just couldn't get it to match on regexr.com but it's probably because of the positive lookbehind.

Comment won't hurt though :D